tests/common-tests.h

```diff
@@ -4692,6 +4779,51 @@ TEST_CASE_METHOD(common_tests, "CLOB", "[core][clob]")
     CHECK(s2.value == s1.value);
 }
 
+TEST_CASE_METHOD(common_tests, "CLOB vector", "[core][clob][vector]")
+{
+    soci::session sql(backEndFactory_, connectString_);
+
+    if (sql.get_backend_name() != "firebird" &&
+        sql.get_backend_name() != "odbc")
+    {
+        WARN("Vector of long strings not supported by the database, skipping the test.");
```

Comment:
What currently happens if we do try to use them with PostgreSQL/Oracle? Do we just get an error or ...?